bridge/commands/handler.go

```diff
@@ -55,6 +55,11 @@ type FullHandler struct {
 	RequiresPortal bool
 	RequiresLogin  bool
 
+	RequiresMatrixLogin            bool
```

Comment:
This shouldn't exist as it turns out people don't like that help output changes based on login state (mautrix/discord#18 / https://github.com/mautrix/go/commit/d1b5e5b49214355f858f2ec7b85c2b7771ed3e91)